examples/action_slackpostmessage.tf

```diff
@@ -4,8 +4,8 @@ resource "humio_action" "example_slackpostmessage" {
   type       = "SlackPostMessageAction"
 
   slackpostmessage {
-    api_token  = "abcdefghij1234567890"
-    channels = ["#alerts", "#ops"]
+    api_token = "abcdefghij1234567890"
+    channels  = ["#alerts", "#ops"]
```

Comment:
```suggestion
    channels  = ["#alerts", "#ops"]

```

:man_shrugging:

Reply:
You're right. The removal of trailing spaces was part of `terraform fmt`.

Humio has started changing "24h" to "1d" on their end, so whenever I deployed one of the example resources, Terraform would tell me that my (unchanged) resource would be updated in-place with ` ~ start = "1d" -> "24h"`. So I fixed that.

It's new behavior in Humio's API, which is why I just fixed it now.